src/main/java/com/internet/shop/controllers/user/DeleteUsersController.java

```diff
@@ -24,8 +23,8 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         String userId = req.getParameter("id");
         Long id = Long.valueOf(userId);
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(id);
-        shoppingCartService.delete(shoppingCart.getId());
+        /* ShoppingCart shoppingCart = shoppingCartService.getByUserId(id);
+        shoppingCartService.delete(shoppingCart.getId());*/
```

Comment:
why have you commented this?
your shopping cart should be deleted with the user